tools/Makefile

```diff
@@ -99,6 +100,7 @@ $(curdir)/cmake/compile += $(curdir)/libressl/compile $(curdir)/ninja/compile $(
 $(curdir)/dosfstools/compile := $(curdir)/automake/compile
 $(curdir)/e2fsprogs/compile := $(curdir)/libtool/compile $(curdir)/util-linux/compile
 $(curdir)/elfutils/compile := $(curdir)/m4/compile $(curdir)/zlib/compile $(curdir)/gnulib/compile $(curdir)/libtool/compile
+$(curdir)/erofs-utils/compile := $(curdir)/libtool/compile $(curdir)/xz/compile $(curdir)/lz4/compile
```

Comment:
For libuuid, just add `$(curdir)/util-linux/compile` at the end

Reply:
thanks! will try!

Reply:
done, I'm trying to build locally (it may need some time), but maybe retry the CI as well? @robimarko

Reply:
It seems the CI issue is resolved.